rotkehlchen/tests/unit/accounting/test_decoded_events.py

```diff
@@ -0,0 +1,200 @@
+# Some utility timestamps. They are used for mocked prices.
+import pytest
+
```

Comment:
Also this filename does not make much sense. Is it just for curve? Seems like it. For EVM transactions we also have `tests/unit/accounting/test_transactions.py`. What I would suggest is make a new directory under `unit/accounting` and call it `unit/accounting/evm` and put both files there. Rename this one to curve, since it concerns only curve events.

So would be something like:

```
tests/unit/accounting/evm/
 |
 | --- test_transactions.py
 | --- test_curve.py
```